module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+namespace
+{
+constexpr size_t width = 32; // 240;
+
+constexpr size_t height = 32; // 160;
+
+using VectorPair = std::vector<std::pair<uint8_t, uint8_t> >;
```

Comment:
Why there is a space between > and between < there is no one? It is inconsitent.